test/check-application

```diff
@@ -179,11 +179,16 @@ class TestApplication(testlib.MachineCase):
         b.wait_present("#containers-containers .container-name:contains('a')")
         b.wait_present("#containers-containers .container-name:contains('b')")
 
+        if self.machine.image in [ "rhel-8-3" ]:
```

Comment:
Would be so nice, if we could use this one: https://github.com/cockpit-project/cockpit/commit/58d1d00d3dcce89ddcdd0a1f5f585f43e7382d0e :) (needs a new `test/common`)

Reply:
Yes, we should probably add some convenience functions for gaining/dropping privs to testlib and hide all the complexity there.

Reply:
Note that 8.4 also has the old cockpit still; there is some trouble with the erratum creation.

Reply:
The tests don't seem to agree: they pass without special-casing rhel-8-4.  Does that make sense?

Reply:
Heh, did they maybe all get skipped before getting to this point?

Reply:
So, yeah, let's just use the approach in https://github.com/cockpit-project/cockpit/pull/14981.  Unfortunately, it's not released yet... should we just get test/common from 58d1d00?

Reply:
> should we just get test/common from 58d1d00?

That doesn't work because git doesn't allow fetching by SHA.. silly me.

Reply:
Ok, done anyway!